lib/filter.js

```diff
@@ -36,5 +36,19 @@ function filterPullRequest(pull_request, log) {
     return true;
 }
 
+function filterPullRequestByLabel(pull_request, label, log) {
```

Comment:
This is now dead code I believe (since we aren't dealing with 'do not merge'); please remove it.